### PR TITLE
Split new module: azure-tools-common

### PR DESCRIPTION
**Pull request by andxu**

### CODE REVIEW

File: azure-tools-common/pom.xml [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-junit4</artifactId>
+            <version>${powermock.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    
+</project>
```
Comment (Flanker32) [inline azure-tools-common/pom.xml]: Please add a new line at the end

File: azure-tools-common/pom.xml [hunk 2]
```diff
@@ -0,0 +1,183 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-tools-common</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
```
Comment (Flanker32) [inline azure-tools-common/pom.xml]: include parent pom?

File: azure-tools-common/pom.xml (hunk 1 above)
Comment (andxu) [reply azure-tools-common/pom.xml]: done

File: azure-tools-common/pom.xml (hunk 2 above)
Comment (andxu) [reply azure-tools-common/pom.xml]: done

**Pull request merged**